src/test/java/ca/corefacility/bioinformatics/irida/ria/unit/web/analysis/AnalysesListingServiceTest.java

```diff
@@ -56,8 +58,8 @@ public void testGetPagedSubmissionsForAdmin() throws IridaWorkflowNotFoundExcept
 				ImmutableMap.of());
 
 		when(analysisSubmissionService
-				.listAllSubmissions(eq(searchValue), any(String.class), eq(null), Matchers.any(),
-						Matchers.any())).thenReturn(AnalysesDataFactory.getPagedAnalysisSubmissions());
+				.listAllSubmissions(eq(searchValue), isNull(), isNull(), isNull(),
```

Comment:
For the any(String.class) here and in tests below why are we not checking for isNull()?

Reply:
Mockito 1.x which we were on would accept `null` values for `any()` even if a class if expected.